utils/cq/requirements.txt

```diff
@@ -11,4 +11,5 @@ isort
 paramiko
 pyenchant
 pylint
+toml
```

Comment:
ah good point. This is the problem. We don't run the exact same env.

Reply:
Right. I think the discrepancy here is a different env

Reply:
Yes, that's the problem. Let's disable isort for now until we figure out better solution. Everybody is running a different env. Going forward, I would suggest moving these code quality stuff to a GHA which runs before all other tests. That way we have a unified env for code quality.

Reply:
isort already provides an official GHA: https://pycqa.github.io/isort/docs/configuration/github_action.html

Reply:
I think definitely a GHA for this would be good. Then, a githook is just a convenience to catch before pushing.